test/ArgTools.jl.faceup

```diff
@@ -0,0 +1,338 @@
+«k:module» ArgTools
+
+«k:export»
+    arg_read,  ArgRead,  arg_readers,
+    arg_write, ArgWrite, arg_writers,
+    arg_isdir, arg_mkdir, «:julia-ts-macro-face:@arg_test»
+
+«k:import» Base: AbstractCmd, CmdRedirect, Process
+
+«k:if» isdefined(Base.Filesystem, «:julia-ts-quoted-symbol-face::prepare_for_deletion»)
+    «k:using» Base.Filesystem: prepare_for_deletion
+«k:else»
+    «k:function» «f:prepare_for_deletion»(path::«t:AbstractString»)
+        «k:try» prepare_for_deletion_core(path)
+        «k:catch»
+        «k:end»
+    «k:end»
+    «k:function» «f:prepare_for_deletion_core»(path::«t:AbstractString»)
+        isdir(path) || «k:return»
+        chmod(path, filemode(path) | «c:0o333»)
+        «k:for» name «k:in» readdir(path)
+            «v:path′» = joinpath(path, name)
+            prepare_for_deletion_core(path′)
+        «k:end»
+    «k:end»
+«k:end»
+
+«k:const» «v:nolock_docstring» = «s:"""
+Note: when opening a file, ArgTools will pass `lock = false` to the file `open(...)` call.
+Therefore, the object returned by this function should not be used from multiple threads.
+This restriction may be relaxed in the future, which would not break any working code.
+"""»
+
+«k:if» VERSION ≥ «s:v"1.5"»
+    «f:open_nolock»(args...; kws...) = open(args...; kws..., «:julia-ts-quoted-symbol-face:lock»=«c:false»)
```

Comment:
It doesn't seem like `julia-ts-quoted-symbol-face` is an appropriate font-lock for a keyword argument.

Reply:
This is my reasoning behind choosing this face for keyword argument names. The name of a keyword argument is a symbol. Hence, it is font-locked with the same face as a quoted symbol `:symbol` is.

Which face would you recommend?

Reply:
I don't see this as any more symbol-like than any other token in a julia program. I would recommend not font-locking this at all or if you wanted to, then font-lock using a new `julia-ts-kwarg-face` that the user can set to inherit from whatever they want.

Reply:
Good idea, I did just that!